bindata/daemon/04.daemonset.yaml

```diff
@@ -26,6 +26,7 @@ spec:
         image: {{.DpuOperatorDaemonImage}}
```

Comment:
this image no longer gets picked up?

I had to hard code the value for local testing...
i.e.
``{{.DpuOperatorDaemonImage}}`` -> ``localhost:5001/dpu-daemon:dev``